### PR TITLE
Better: Add /1.0/users/:user_id/sources_permissions documentation. RD-29591

### PR DESCRIPTION
**Pull request by DrissTM**
**Ticket(s)**:
- [RD-29591](https://jira.ringcentral.com/browse/RD-29591)

**Change(s)**:
- Add doc about `/1.0/users/:user_id/sources_permissions`

**ToDo(s)**:
- [x] Generate file

### CODE REVIEW

**Comment by DrissTM** [comment]
@brenauj could you checkout my branch and generate the files please? 🙏

File: specs/engage-digital_openapi3.yaml [hunk 1]
```diff
@@ -6994,6 +6996,82 @@ paths:
...
+              - approval_threshold_30
+              - approval_threshold_40
+              - approval_threshold_50
+              - approval_threshold_60
+              - approval_threshold_70
+              - approval_threshold_80
+              - approval_threshold_90
+              - approval_threshold_100
+          in: query
+          name: source_x_id
```
Comment (brenauj) [inline specs/engage-digital_openapi3.yaml:7061]:
```suggestion
          name: "{source_x_id}"

```
We should make sure that people understand that this name is a placeholder and should be a content source id.

We should also add an example key for this param.

**Review by brenauj — CHANGES_REQUESTED** [review]

File: specs/engage-digital_openapi3.yaml (hunk 1 above)
Comment (DrissTM) [reply specs/engage-digital_openapi3.yaml]: done in [dd29655](https://github.com/ringcentral/engage-digital-api-docs/pull/248/commits/dd296559e0aad011ab255346b51317e44477b2ab)

**Review by brenauj — APPROVED** [review]
LGTM :+1:

**Pull request merged**